src/main/java/net/dv8tion/jda/api/events/user/update/UserUpdateFlagsEvent.java

```diff
@@ -25,45 +40,45 @@
  * member was updated and gives us the updated member object. In order to fire a specific event like this we
  * need to have the old member cached to compare against.
  */
-public class UserUpdateFlagsEvent extends GenericUserUpdateEvent<Integer>
+public class UserUpdateFlagsEvent extends GenericUserUpdateEvent<EnumSet<User.UserFlag>>
 {
     public static final String IDENTIFIER = "public_flags";
     
-    public UserUpdateFlagsEvent(@Nonnull JDA api, long responseNumber, @Nonnull User user, int oldFlags)
+    public UserUpdateFlagsEvent(@Nonnull JDA api, long responseNumber, @Nonnull User user, EnumSet<User.UserFlag> oldFlags)
     {
-        super(api, responseNumber, user, oldFlags, User.UserFlag.getRaw(user.getFlags()), IDENTIFIER);
+        super(api, responseNumber, user, oldFlags, user.getFlags(), IDENTIFIER);
     }
 
     /**
-     * @return The old {@code Integer} representation of the {@link net.dv8tion.jda.api.entities.User.UserFlag UserFlags}.
+     * @return The old {@code EnumSet<{@link net.dv8tion.jda.api.entities.User.UserFlag UserFlag}>} representation of the User's flags.
      */
     @Nullable
-    public Integer getOldFlags()
+    public EnumSet<User.UserFlag> getOldFlags()
     {
         return getOldValue();
     }
 
     /**
-     * @return Possibly-null EnumSet of previous {@link net.dv8tion.jda.api.entities.User.UserFlag UserFlags}.
+     * @return The old raw {@code Integer} value of the {@link net.dv8tion.jda.api.entities.User.UserFlag UserFlags}.
      */
     @Nullable
-    public EnumSet<User.UserFlag> getOldFlagSet(){
-        return previous == null ? null : User.UserFlag.getFlags(previous);
+    public Integer getOldFlagsRaw(){
```

Comment:
Change this to `int` and remove the `Nullable`

Reply:
I assume for both methods?
I changed it